dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.submitDealCancellation.test.ts

```diff
@@ -28,15 +29,15 @@ const auditDetails = generateTfmAuditDetails(tfmUserId);
 
 describe('tfm-deals-cancellation-repo', () => {
   const updateOneMock = jest.fn();
+  const updateManyMock = jest.fn();
+  const findMock = jest.fn();
+  const findToArrayMock = jest.fn();
   const getCollectionMock = jest.fn();
 
-  beforeAll(() => {
-    jest.useFakeTimers();
-  });
+  const matchingFacilityId1 = new ObjectId();
+  const matchingFacilityId2 = new ObjectId();
 
-  afterAll(() => {
-    jest.useRealTimers();
-  });
```

Comment:
Accidentally two sets of using fake timers was added to this file, so removing one set here (further down in the file we still do this)